lib/node_modules/@stdlib/random/base/mt19937/lib/factory.js

```diff
@@ -80,18 +116,18 @@ function factory( seed ) {
 			i = 1;
 			j = 0;
 			k = max( N, seed.length );
-			state = 19650218 | 0; // asm type annotation
+			state = 19650218 >>> 0; // asm type annotation
 		} else {
 			if ( !isPositiveInteger( seed ) ) {
 				throw new TypeError( 'invalid argument. Must provide a positive integer. Value: `' + seed + '`.' );
 			}
 			if ( seed > MAX_SEED ) {
-				throw new RangeError( 'invalid argument. Must provide a positive integer less than the maximum signed 32-bit integer. Value: `' + seed + '`.' );
+				throw new RangeError( 'invalid argument. Must provide a positive integer less than the maximum unsigned 32-bit integer. Value: `' + seed + '`.' );
```

Comment:
less than or equal to

Reply:
Sorry. Disregard this. Although why is the max seed `UINT32_MAX - 1`? Why cannot it not be `UINT32_MAX`?

Reply:
Sorry, I overlooked the fact that `UINT32_MAX` is `2^{32} - 1` and not `2^{32}`.

Reply:
No worries. Thanks for clarifying!